src/core/lib/debug/trace.h

```diff
@@ -111,6 +112,472 @@ class SavedTraceFlags {
   std::map<std::string, bool> values_;
 };
 
+extern TraceFlag api_trace;
+#define GRPC_API_TRACE_LOG(level) \
```

Comment:
Instead of defining a separate macro for each tracer, how about just defining it once, such that the tracer can be passed in as a parameter:

```
#define GRPC_TRACE_LOG(tracer, level) \
    LOG_IF(level, GRPC_TRACE_FLAG_ENABLED(tracer))
```

Note that we do want to use `GRPC_TRACE_FLAG_ENABLED()` here instead of directly calling `tracer.enabled()`, as per the comment on line 68.

Reply:
I had the same thought! Done.